packages/style/scss/components/tooltip.scss

```diff
@@ -42,10 +43,11 @@
         padding: 0 var(--arrow-size);
 
         .tooltip-arrow {
+            top: 50%;
             left: 0;
-            margin-top: calc(-1 * var(--arrow-size));
-            border-width: var(--arrow-size) var(--arrow-size) var(--arrow-size) 0;
-            border-right-color: var(--tooltip-color);
+            margin-top: -5px;
+            border-width: 5px 5px 5px 0;
+            border-right-color: rgba(var(--navy-blue-60-rgb), 0.94);
```

Comment:
Only adding `top: 50%` fixes the issue, please revert the rest we would like to keep those calculations
![image](https://user-images.githubusercontent.com/35579930/175052175-6e6e057e-029c-4e38-bb33-03fcbac2bf2f.png)

Reply:
Gotcha, thanks! I added the respective percentages back and left the calculations - I was able to observe no visible changes and it indeed fixes the issues found in admin-ui 👍